python/lsst/ap/association/packageAlerts.py

```diff
@@ -186,24 +190,49 @@ def createDiaSourceBBox(self, bboxSize):
             bbox = geom.Extent2I(bboxSize, bboxSize)
         return bbox
 
-    def createCcdDataCutout(self, cutout, skyCenter, photoCalib):
-        """Convert a cutout into a calibrate CCDData image.
+    def createCcdDataCutout(self, image, skyCenter, bbox, photoCalib, srcId):
+        """Grab an image as a cuttout and return a calibrated CCDData image.
 
         Parameters
         ----------
-        cutout : `lsst.afw.image.ExposureF`
-            Cutout to convert.
+        image : `lsst.afw.image.ExposureF`
+            Image to pull cutout from.
         skyCenter : `lsst.geom.SpherePoint`
             Center point of DiaSource on the sky.
+        bbox : `lsst.geom.Extent2I`
```

Comment:
Isn't it a `BBox` and not and `Extent2I`?

Reply:
It is an Extent2I. Do you want me to change the variable names?

Reply:
Yes please.